### PR TITLE
Add check. If control flexberry-groupedit is used (modelProjection ha…

### PR DESCRIPTION
**Pull request by kafnevod**
…s property caption) userSettings don't implemented

### CODE REVIEW

File: addon/components/object-list-view.js [hunk 1]
```diff
@@ -502,6 +502,10 @@ export default FlexberryBaseComponent.extend(
     }
 
     let cols = this._generateColumns(projection.attributes);
+    if ('caption' in projection) {  //GroupEdit has caption values, dont use userSettings, return basic order
```
Comment (hvostya) [inline addon/components/object-list-view.js]: We write each comment on separate line.
There should be a dot at the end of the comment and a space after "//".

Comment (hvostya) [inline addon/components/object-list-view.js]: Can you add more detailed comment what condition "('caption' in projection)" does?

File: addon/components/object-list-view.js [hunk 2]
```diff
@@ -502,6 +502,10 @@ export default FlexberryBaseComponent.extend(
...
+      return cols;
```
Comment (hvostya) [inline addon/components/object-list-view.js]: Is situation when there are two GroupEdits with the same property names tested?

**Comment by hvostya** [comment]
You should merge with develop in order to repair build.

File: addon/components/object-list-view.js (hunk 2 above)
Comment (kafnevod) [inline addon/components/object-list-view.js]: Tested
_generateColumns generate colums list for every modelProjection

**Pull request merged**